lib/src/core/services/groups_service.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:convert';
+
+import 'package:isar/isar.dart';
+import 'package:kana_to_kanji/src/core/constants/alphabets.dart';
+import 'package:kana_to_kanji/src/core/models/group.dart';
+import 'package:kana_to_kanji/src/core/services/api_service.dart';
+import 'package:kana_to_kanji/src/locator.dart';
+import 'package:http/http.dart' as http;
+
+class GroupsService {
+  final Isar _isar = locator<Isar>();
+  final ApiService _apiService = locator<ApiService>();
+
+  /// Get all the groups related to the alphabet given in parameter.
+  Future<List<Group>> getGroups(Alphabets alphabet,
+      {bool reload = false}) async {
+    final groups = _isar.groups.where().alphabetEqualTo(alphabet).findAll();
+
+    if (reload || groups.isEmpty) {
+      deleteAll();
+      return loadCollection().then(
+          (_) => _isar.groups.where().alphabetEqualTo(alphabet).findAll());
+    }
```

Comment:
Oh, I would not do that 😅 
We should not have the option to do that here. Having a `clear` or `reset` function why not? But if we need to reset the database we would probably go with the [close function of Isar](https://pub.dev/documentation/isar/latest/isar/Isar/close.html).

Reply:
Pourtant tu avais quelque chose de similaire avant